UPGRADING.md

```diff
@@ -1,6 +1,19 @@
 Upgrading Slack-Ruby-Client
 ===========================
 
+### Upgrading to >= 0.16.0
+
+As of 0.16.0, we no longer depend on `activesupport` as a dependency. This means that if you were relying on `activesupport` functionality unintentionally, certain things in you code may break. To fix this, you have two options:
```

Comment:
Let's say this simpler

As of 0.16.0 `slack-ruby-client` no longer requires `activesupport`. If you are using activesupport, add `gem 'activesupport' to your Gemfile.`.